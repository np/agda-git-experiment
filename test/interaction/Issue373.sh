OBJECT_FILE=Imports/Nat.o

if [ -e "$OBJECT_FILE" ]; then
    rm "$OBJECT_FILE"
fi

echo 'ioTCM "Issue373.agda" None (cmd_compile MAlonzo "Issue373.agda" [])' | \
  agda --interaction 2>&1

./Issue373
