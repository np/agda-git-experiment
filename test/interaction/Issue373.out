Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue373 (Issue373.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue373.\n" t)
(agda2-info-action "*Type-checking*" "Compiling Issue373 in Issue373.agdai to MAlonzo/Code/Issue373.hs\n" t)
(agda2-info-action "*Type-checking*" "Calling: ghc -O -o Issue373 -Werror -i/home/nad/research/dtp/Agda2/test/interaction -main-is MAlonzo.Code.Issue373 MAlonzo/Code/Issue373.hs --make -fwarn-incomplete-patterns -fno-warn-overlapping-patterns\n" t)
(agda2-info-action "*Type-checking*" "[1 of 3] Compiling MAlonzo.RTE      ( MAlonzo/RTE.hs, MAlonzo/RTE.o )\n" t)
(agda2-info-action "*Type-checking*" "[2 of 3] Compiling Imports.Nat      ( Imports/Nat.hs, Imports/Nat.o )\n" t)
(agda2-info-action "*Type-checking*" "[3 of 3] Compiling MAlonzo.Code.Issue373 ( MAlonzo/Code/Issue373.hs, MAlonzo/Code/Issue373.o )\n" t)
(agda2-info-action "*Type-checking*" "Linking Issue373 ...\n" t)
(agda2-status-action "Checked")
(agda2-info-action "*Compilation result*" "The module was successfully compiled." nil)
((last . 1) . (agda2-goals-action '()))
Agda2> ok
